k8s-lens.hcl

```diff
@@ -0,0 +1,14 @@
+description = "K8S Lens - Kubernetes IDE/console."
+binaries = ["opt/Lens/lens"]
+
+version "5.3.4" {
+  platform darwin amd64 {
+    source = "https://api.k8slens.dev/binaries/Lens-${version}-latest.20220120.1.dmg"
```

Comment:
Don't worry about it for now, but for future reference you can use `vars` to DRY annoying URLs like this. See [clang-format](https://github.com/cashapp/hermit-packages/blob/master/clang-format.hcl) for an example.

Reply:
eg. you might have:

```hcl
version "5.3.4" {
vars = {
  "date": "20220120",
}
}

platform darwin amd64 {
    source = "https://api.k8slens.dev/binaries/Lens-${version}-latest.${date}.1.dmg"
}


```